Sources/Shared/Notifications/NotificationCommands/NotificationsCommandManager.swift

```diff
@@ -119,8 +119,9 @@ private struct HandlerUpdateWidgets: NotificationCommandHandler {
         Current.Log.verbose("Reloading widgets triggered by notification command")
         return Promise<Void> { seal in
             DispatchQueue.main.async {
-                WidgetCenter.shared.reloadTimelines(ofKind: WidgetsKind.gauge.rawValue)
-                WidgetCenter.shared.reloadTimelines(ofKind: WidgetsKind.details.rawValue)
+                #if os(iOS)
```

Comment:
I think I asked you to add this check right? If so, my bad, we also need to reload in case it is macOS, so this can be removed